contracts/wormhole/contracts/mapping-token/interfaces/IErc721AttrSerializer.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.10;
+
+interface IErc721AttrSerializer {
+    function name() external view returns (string memory);
+    function symbol() external view returns (string memory);
+    function tokenURI(uint256 tokenId) external view returns (string memory);
+    function Serialize(uint256 id) external view returns(bytes memory);
+    function Deserialize(uint256 id, bytes memory data) external;
```

Comment:
Why is it designed to be capitalized for first letter?

`Serialize` vs `serialize`?